kolibri/plugins/facility/assets/src/views/DataPage/SyncInterface/index.vue

```diff
@@ -173,7 +173,10 @@
     },
     methods: {
       managesync() {
-        this.$router.push({ name: PageNames.MANAGE_SYNC_SCHEDULE });
+        this.$router.push({
+          name: SyncPageNames.MANAGE_SYNC_SCHEDULE,
+          params: { facilityId: this.facility.id },
```

Comment:
Within the facility plugin, the route parameter is `facility_id`. I think we should just standardize on that for the route parameters to avoid this ongoing confusion.